content/2_3_riak.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{The Riak Key-Value Store}
+\chapterauthor{Daniel Rutz and Paul Thore Flachbart}
+
+\section{Introduction}
```

Comment:
And subsections, respectively.

Reply:
Resolved, please check for any forgotten section.